docker/ftest/src/test/resources/tomcat/Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM tutum/tomcat:7.0
-
+FROM tomcat:jre8
```

Comment:
@bartoszmajsak you mean tag this image & push to arquillian dokcerhub account & use it from there.